xwiki-rendering-api/src/main/java/org/xwiki/rendering/block/ListBLock.java

```diff
@@ -24,7 +24,9 @@
  *
  * @version $Id$
  * @since 1.5M2
+ * @deprecated since 8.1M1, use {@link ListBlock} instead
  */
+@Deprecated
 public interface ListBLock extends Block
```

Comment:
As I indicated you need to extend the new ListBlock here. It automatically fix all implementations of ListBLock, even those you don't know.